plugins/nf-amazon/src/resources/META-INF/extensions.idx

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2013-2024, Seqera Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+nextflow.cloud.aws.batch.AwsBatchExecutor
+nextflow.cloud.aws.util.S3PathSerializer
+nextflow.cloud.aws.util.S3PathFactory
+nextflow.cloud.aws.fusion.AwsFusionEnv
+nextflow.cloud.aws.mail.AwsMailProvider
+nextflow.cloud.aws.scm.S3RepositoryFactory
```

Comment:
This should not be needed anymore using the new plugin system. Consider synching this branch with master